docs/Files/UI/Solution-Manager/Library/EscalationManager/Escalation-Manager.md

```diff
@@ -0,0 +1,17 @@
+---
+lang: en-us
+title: Managing Escalation Manager
+viewport: width=device-width, initial-scale=1.0
+---
+
+# Overview
+
+Escalation Manger has options related to escalations.
+
+![Managing Escalation Manger](../../../../../Resources/Images/SM/Library/EscalationManager/EscalationManager.png "Threshold Grid")
+
+:::note
+Use the bar on the left side of the screen
+:::
+
+Please check back for more content
```

Comment:
We should end all sentences with punctuation.

Reply:
Please review all pages for spelling issues -- 'Manager' has been misspelled as 'Manger' in several locations.

Reply:
Added punctuation (a period .), fixed Manager spelling